### PR TITLE
[Doc Improvement][Suggested actions note update][3281361]

### PR DESCRIPTION
**Pull request by v-rmvv**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/conversation-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8a9e529d04fdc96d20f91231adc4229715a033dd/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fconversation-messages.md) | 91 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/58f30e03-13be-4225-a598-d55bfc52e1c9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/conversations/conversation-messages.md [hunk 1]
```diff
@@ -594,7 +594,8 @@ The following illustrates an example of suggested actions:
 
 > [!NOTE]
 >
-> * `SuggestedActions` are only supported for one-on-one chat bots and text based messages and not for Adaptive Cards or attachments.
+> * `SuggestedActions` are only supported for one-on-one chat bots with both text based messages and Adaptive Cards.
+> * `SuggestedActions` aren't supported for one-on-one chat bots with attachments.
```
Comment (limamicro) [inline msteams-platform/bots/how-to/conversations/conversation-messages.md:598]: How about we also add "and other conversation types" after "attachments".
Right now, Suggested action only work in 1:1 with text message and adaptive card, for all other scenarios, it does not support.

**Review by limamicro — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/conversation-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b85d1ce6cf605a47f8d1b0016d73039638f43cd9/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fconversation-messages.md) | 91 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/626a3287-6f2c-4b71-b14e-6fe4e1add38a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by limamicro — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/conversation-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c129b70c883a2c3fb5a8d491d3938a52925f54f2/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fconversation-messages.md) | 91 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c698831-1511-48aa-9db1-d91021fb6ade) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/conversation-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c3c01ee5b8a8ddc4933a5c0436432dfb99c77d3/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fconversation-messages.md) | 91 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/764c575d-8f80-499d-a6bb-fc881f4c7917) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/conversations/conversation-messages.md (hunk 1 above)
Comment (v-rmvv) [reply msteams-platform/bots/how-to/conversations/conversation-messages.md]: Updated.

**Review by limamicro — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/conversation-messages.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cfdc4ef95a17f925febdd4a69f588c203bbfb291/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fconversation-messages.md) | 91 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/db0a34d8-6fbf-4e4c-a9f6-9bd25a14dde6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**